public/app/features/templating/template_srv.ts

```diff
@@ -284,7 +284,10 @@ export class TemplateSrv implements BaseTemplateSrv {
 
     if (!variable) {
       if (macroRegistry[variableName]) {
```

Comment:
Nit: couldn't we remove this line? Since we are checking that we actually got a macro on line 288?

Reply:
@mckn good catch, it's a merge issue, not supposed to look like this!